### PR TITLE
Remote development quick start.

### PR DESCRIPTION
**Pull request by pswaine**

### CODE REVIEW

File: content/quick-start/remote-development-using-a-remote-environment.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    weight: 110
+
+weight: 110
+---
+After [creating a remote environment]({{< relref "remote-development-creating-a-remote-environment.md" >}}) you'll need to open the project directory in a terminal, then create or switch to the branch specified when generating the token. The `init` command can then be executed: 
+
+```bash
+cd hello-world
+git checkout -b cpdev/pswaine
+cp-remote init NjBkZmExOWEtZTlhZS00MmE1LWJhZTktNDRlZTM4OGQwOWIyLGUwMzhhMTUxLTQ0MTEtNDc2MS1hMjBlLWQxNGU3MDcwNDBhMSw1ZmYzMjJlMC0wODE4LTExZTctYWQwMC0wYTU4MGE4NDA0MDQscHN3YWluZSxjcGRldi9wc3dhaW5l
```
Comment (sroze) [inline content/quick-start/remote-development-using-a-remote-environment.md]: That contains your API key, it probably shouldn't be there 😜

File: content/quick-start/remote-development-using-a-remote-environment.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    parent: 'quick-start'
+    weight: 110
+
+weight: 110
+---
+After [creating a remote environment]({{< relref "remote-development-creating-a-remote-environment.md" >}}) you'll need to open the project directory in a terminal, then create or switch to the branch specified when generating the token. The `init` command can then be executed: 
+
+```bash
+cd hello-world
+git checkout -b cpdev/pswaine
```
Comment (sroze) [inline content/quick-start/remote-development-using-a-remote-environment.md]: You don't need that, you shouldn't never have to checkout this branch locally actually.

File: content/quick-start/remote-development-using-a-remote-environment.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Remote Development: Using a Remote Environment"
+menu:
+  main:
+    parent: 'quick-start'
+    weight: 110
+
+weight: 110
+---
+After [creating a remote environment]({{< relref "remote-development-creating-a-remote-environment.md" >}}) you'll need to open the project directory in a terminal, then create or switch to the branch specified when generating the token. The `init` command can then be executed:
```
Comment (sroze) [inline content/quick-start/remote-development-using-a-remote-environment.md]: > then create or switch to the branch specified when generating the token

nop, it's done by `cp-remote`

Comment (sroze) [inline content/quick-start/remote-development-using-a-remote-environment.md]: Actually it's not even done. Your current HEAD commit (whichever the branch you're in just right now) will be forced push to this `cpdev/...` branch by `cp-remote` without checking it out.

**Comment by tonypiper** [comment]
worth also linking these into http://35.186.240.149/remote-development/ - wouldn't be obvious that they were in the quick start section.

**Comment by sroze** [comment]
🚀 

Nice @pswaine 👍

**Pull request merged**